### ISSUE TITLE
Accessibility description is missing when using widget

### ISSUE THREAD

**Issue by warcarr** [timusus/Shuttle#72]
**Shuttle version:**
v1.6.5-beta3

**Device:**
Nexus 6P OS:Android Nougat 7.1.2

**Description of bug:** 
Buttons in the widget don't have content description

 **Steps to reproduce:**
Create a widget, then launch Talkback, and you would find out the buttons aren't labeled.

**Expected outcome:**
Expected outcome is for a Talkback user to know what button plays the music, skips to the next track or jump back to the previous track.

**Observations (actual outcome):**
The feedback i get from the TTS in conjunction with the screen reader is "button, button."*Hi,

The current beta has fixed the lack of content description from within the Notification bar, but I have discovered another area in Shuttle music, where the buttons are not labeled, and that is when one uses a widit.

Would you kindly look into that issue?

Thank you.
